content/en/blog/_posts/2025-01-13-introducing-jobset/2025-01-13-introducing-jobset.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: blog
+title: "Introducing JobSet"
+date: 2024-01-13
```

Comment:
@sftim we missed the date; what new date do you suggest we put here?

Reply:
Ah, sorry about that.

@danielvegamyhre can you add `draft: true` into the front matter and a new future & weekday publication date? Then we can merge it as a draft and the process to fix things up is simpler.

Reply:
> Ah, sorry about that.
> 
> @danielvegamyhre can you add `draft: true` into the front matter and a new future & weekday publication date? Then we can merge it as a draft and the process to fix things up is simpler.

Sure, done. I chose Feb 3rd this time to give us some breathing room.